pkg/ottl/ottlfuncs/README.md

```diff
@@ -25,23 +25,23 @@ Functions
 
 ## Concat
 
-`Concat(delimiter, ...values)`
+`Concat(delimiter, values[])`
```

Comment:
I feel like this would be more conventional with how other languages do string joining if the function signature were `Concat(values[], delimiter)`, but I would be curious to hear what others think.

Reply:
Agreed